### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by TrustHenry**

### CODE REVIEW

File: src/Stoa.ts [hunk 1]
```diff
@@ -147,7 +147,7 @@ class Stoa extends WebService
         this.app.get("/block-transactions", this.getBlockTransactions.bind(this));
         this.app.get("/boa-stats", this.getBOAStats.bind(this));
         this.app.get("/spv/:hash", this.verifyPayment.bind(this));
-        this.app.get("/coinmarketcap", this.getcoinMaketCap.bind(this));
+        this.app.get("/coinmarketcap", this.getcoinMarketCap.bind(this));
```
Comment (Geod24) [inline src/Stoa.ts]:
```suggestion
        this.app.get("/coinmarketcap", this.getCoinMarketCap.bind(this));
```
?

File: src/modules/coinmarket/CoinGeckoMarket.ts [hunk 2]
```diff
@@ -26,9 +26,9 @@ export interface CoinMarket {
     fetch(): Promise<any>;
 }
 
-export class CoinGeckoMaket implements CoinMarket {
+export class CoinGeckoMarket implements CoinMarket {
     /**
-     * Market Client i.e Gecko_coin_market 
+     * Market Client i.e Gecko_coin_market
```
Comment (kchulj) [inline src/modules/coinmarket/CoinGeckoMarket.ts:31]: are the underscores necessary?

File: src/modules/service/CoinMarketService.ts [hunk 3]
```diff
@@ -58,10 +58,10 @@ export class CoinMarketService {
     }
 
     /**
-     * This method prefrom bosagora coin market cap recovery and handle 
+     * This method prefrom bosagora coin market cap recovery and handle
```
Comment (kchulj) [inline src/modules/service/CoinMarketService.ts]:
```suggestion
     * This method performs BOSAGORA coin market cap recovery and handle
```

File: src/Stoa.ts (hunk 1 above)
Comment (TrustHenry) [reply src/Stoa.ts]: Thanks, I fixed it now. (Camel case)

File: src/modules/coinmarket/CoinGeckoMarket.ts (hunk 2 above)
Comment (TrustHenry) [reply src/modules/coinmarket/CoinGeckoMarket.ts:31]: Editor trolling removal function. 
Service :)

File: src/modules/service/CoinMarketService.ts (hunk 3 above)
Comment (TrustHenry) [reply src/modules/service/CoinMarketService.ts]: Thanks, I updated now.

File: src/modules/service/CoinMarketService.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
 /*******************************************************************************
 
-    This file cotain implementation of coin market service for boascan. 
+    This file cotain implementation of coin market service for boascan.
```
Comment (kchulj) [inline src/modules/service/CoinMarketService.ts]:
```suggestion
    This file contains the implementation of coin market service for boascan.
```

File: src/modules/service/CoinMarketService.ts [hunk 5]
```diff
@@ -20,30 +20,30 @@ import { Time } from '../common/Time';
...
 
     /**
-     * Job to collect coin market data 
+     * Job to collect coin market data
      */
     private job: cron.ScheduledTask | null = null;
 
     /**
-     * Job exection status 
+     * Job exection status
```
Comment (kchulj) [inline src/modules/service/CoinMarketService.ts]:
```suggestion
     * Job execution status
```

File: src/modules/service/CoinMarketService.ts [hunk 6]
```diff
@@ -58,10 +58,10 @@ export class CoinMarketService {
     }
 
     /**
-     * This method prefrom bosagora coin market cap recovery and handle 
+     * This method prefroms BOSAGORA coin market cap recovery and handle
```
Comment (kchulj) [inline src/modules/service/CoinMarketService.ts]:
```suggestion
     * This method performs BOSAGORA coin market cap recovery and handle
```

**Review by kchulj — CHANGES_REQUESTED** [review]

File: src/modules/service/CoinMarketService.ts (hunk 6 above)
Comment (TrustHenry) [reply src/modules/service/CoinMarketService.ts]: Thanks, I updated now.

**Review by kchulj — APPROVED** [review]
LGTM
TY!

**Pull request merged**